spec/dummy/app/themes/default/assets/stylesheets/thredded.scss

```diff
@@ -1,15 +1,25 @@
+@import url(http://fonts.googleapis.com/css?family=Lato:400,900,400italic);
+
 @import "bourbon";
-@import "base/base";
+@import "base/grid-settings";
 @import "neat";
-@import "thredded/variables";
-@import "thredded/common";
-@import "thredded/layout";
-@import "thredded/nav";
-@import "thredded/sidebar";
-@import "thredded/messageboards";
-@import "thredded/topics/common";
-@import "thredded/topics/public";
-@import "thredded/topics/private";
-@import "thredded/posts";
-@import "thredded/drop_forms";
+@import "base/base";
```

Comment:
yeah i think thats a great idea. now that you mention it i've seen that approach across several js libraries that need css too, like chosen.js